src/config/schema.ts

```diff
@@ -219,6 +219,24 @@ export const configSchema = {
       type: "object",
       properties: {
         initialStatus: { type: "string", enum: Object.values(AccountStatus) },
+        information: {
+          type: "array",
+          items: {
+            type: "object",
+            properties: {
+              name: { type: "string" },
+              type: {
+                type: "string",
+                enum: ["string", "integer", "float", "boolean"],
+                default: "string",
```

Comment:
```suggestion
```

I would suggest not having a default here. ie: it has to be passed explicitly

Reply:
done https://github.com/GaloyMoney/galoy/pull/1465/commits/c536cd781df7aba2f2611bb8675e4ac38188f347#diff-a25ae601baae9bca444c00d66e4934596768a26eafd5208c8fa077dcf31b4ae9L231